### ISSUE TITLE
Submit button not visible on Contact form

### ISSUE THREAD

**Issue by debo19** [layer5io/layer5#2701]
**Current Behavior**

https://user-images.githubusercontent.com/85789734/161622472-409b5fd9-c349-4231-bc48-7e6e17acf7ff.mov

**Desired Situation**
The submit button must be visible on opening the form.

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

**Comment by MukulKolpe** [comment]
Hey @debo19 can I work on this issue?

**Comment by leecalcote** [comment]
@MukulKolpe, yes, as you do, it should be noted that the goal is to move away from the MailChimp-based form and move to a native form like the one you find on https://layer5.io/newcomers. The Contact form fits into a single page, and so, there is no need for the multi-page aspect of the Newcomers form.

**Comment by DurveshKumarPal** [comment]
@debo19 can I work on this issue.

**Comment by debo19** [comment]
@MukulKolpe any updates on this?

**Comment by MukulKolpe** [comment]
Hey, @debo19...sorry I was not around for a few days...had urgent offline exams after the longest of times...Tried to work around this issue in between whenever I had time, but it took more time than I anticipated. Right now it is almost done...will make a PR real soon(max 1 more day)...sorry for taking so much time.
